### DISCUSSION TITLE
How to get access to the new and improved snow2 algorithm?

### DISCUSSION THREAD

**Discussion by enrico-agostini** [PMEAL/porespy#541] [Q&A] [answered]
Hi all,

When trying to use `ps.networks.snow2`, it is not prompted, the only options are shown in the following image.
![snow2](https://user-images.githubusercontent.com/53181559/122907783-c7a29780-d353-11eb-9d26-2789c0c679e6.jpg)
If I try to use it I get an error: `module 'porespy.networks' has no attribute 'snow2'`

I downloaded the module from conda using `conda install -c conda-forge porespy` as recommended.
I have `Python 3.8.5` installed with the latest version of Anaconda.

thanks,
Enrico Agostini

**Comment by ma-sadeghi** [comment, marked as answer]
Hi Enrico, `snow2` is not yet deployed. It's on our `dev` branch. You have two options: either wait for a week or so until we release PoreSpy 2.0, which has `snow2` (then you can do `conda install -c conda-forge porespy`), or clone our repo and use the dev branch. To install the dev version, go to command line and run:

1. `git clone https://github.com/PMEAL/porespy`  # copies the repo in your storage
2. `cd porespy`  # enters the "porespy" directory
3. `pip install -e .`  # installs the development version into your Python environment

Just make sure to uninstall any `porespy` you might already have installed in the environment before executing the above commands.

**Reply by dpereponov** [reply]
Hello, Amin Sadeghi! 
Will other functions be also added with releasing PoreSpy 2.0 (like geometrical tortuosity)?

**Reply by enrico-agostini** [reply]
Thanks Amin you were very helpful, I guess I can wait for some time! Otherwise I'll do as you pointed out.
Have a nice day,
Enrico

**Reply by ma-sadeghi** [reply]
@dpereponov Everything that you can find on our `dev` branch will be on PoreSpy 2.0.
